website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,400 @@
+
+#### Integrating with Operator Lifecycle Manager
+
+The Operator Lifecycle Manager (OLM) is often used to distribute operators via operator catalogs. The operator deployment object in an OLM integrated operator is defined in the `ClusterServiceVersion` yaml. It is important to remember to set the node affinity block in each of the `spec.template.spec` under `spec.install.deployments`.
+
+## Examples
+This section attempts to show how each of the sample operators could be updated to declare a set of supported architectures. Most of the time, these changes will be in `yaml` files, but different kinds of operators embed `PodSpec` and `PodTemplateSpec` objects in different places.
+
+The most common directory with objects that will need an architecture affinity block is the operator's `config` directory. Additionally, if the operator is configured for OLM, there will likely be a deployment object in the `bundle/manifests/*.clusterserviceversion.yaml`. These locations are common across all operator types.
+
+Each of these examples can be in the [OperatorSDK GitHub repository](https://github.com/operator-framework/operator-sdk).
+
+### Ansible Operators
+In the `testdata/ansible` samples app, you'll find several configuration objects that need a nodeAffinity specification in the `config` and `bundle/manifests` directories. The full list is as follows:
+- `testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml`
+- `testdata/ansible/memcached-operator/config/default/manager_config_patch.yaml`
+- `testdata/ansible/memcached-operator/config/default/manager_auth_proxy_patch.yaml`
+- `testdata/ansible/memcached-operator/config/manager/manager.yaml`
+
+The Ansible example also has Pod-defining objects as part of the test directories. For completeness, these should also need to define a nodeAffinity specification. The operator author will know best if tests are expected to be run in mixed-architecture environments, but updating these helps future-proof the operator.
+- `testdata/ansible/memcached-operator/config/testing/debug_logs_patch.yaml`
+- `testdata/ansible/memcached-operator/config/testing/manager_image.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/Always.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/IfNotPresent.yaml`
+- `testdata/ansible/memcached-operator/config/testing/pull_policy/Never.yaml`
+
+ Ansible operators may also create Pod-defining objects dynamically. An example of this kind of object would be in `testdata/ansible/memcached-operator/roles/memcached/tasks/main.yml`.
+ ```yaml
+# tasks file for Memcached
```

Comment:
Why not provides the example direct? I mean, the final result with the config.

Reply:
My thought process was trying to provide a "before" and "after" so that users could follow along to what was actually changing.

Reply:
I think is better if we just add an example with the final result.